.github/workflows/automation-benchmark-tests.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           QA_AWS_ACCOUNT_NUMBER: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}
           suites: benchmark chaos reorg load
       - name: Run Tests
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@fc3e0df622521019f50d772726d6bf8dc919dd38 # v2.3.19
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@65680c51a6f5bdf33ca6686ad806753575c1b12d # v2.3.20
```

Comment:
This should be af92c5fae8dcf1659201e907db82d221fc304b94 # v2.3.21 to use your merged commit instead of the pr branch commit that will get deleted at some unknown point in the future.